<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Frederic Reinier" />
    <meta name="date" content="2019-03-19" />
    <title>Frederic Reinier’s resume</title>

        <link href="test_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
    <link href="test_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
    <link href="test_files/paged-0.2/css/resume.css" rel="stylesheet" />
    <script src="test_files/paged-0.2/js/config.js"></script>
    <script src="test_files/paged-0.2/js/paged.js"></script>
    <script src="test_files/paged-0.2/js/hooks.js"></script>
    
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<p><img src="IMG_0627.JPG" alt="Frederic Reinier" style="width:70.0%" /></p>
<p>RESEARCH SCIENTIST WITH STRONG EXPERIENCE IN NEXT GENERATION SEQUENCING DATA &amp; STATISTICAL ANALYSIS ON CANCER GENOMICS</p>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:fredreinier@gmail.com">fredreinier@gmail.com</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/fred3ric">github.com/fred3ric</a></li>
<li><i class="fa fa-linkedin"></i> <a href="https://www.linkedin.com/in/frederic-reinier/">in/frederic-reinier</a></li>
<li><i class="fa fa-phone"></i> 0615522175</li>
<li>For more information, please contact me via email.</li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>Experienced in statistical analysis, statistical learning models, and optimization methods.</p></li>
<li><p>Full experience with next generation sequencing data analysis.</p></li>
<li><p>Programming: R, Perl, Python, UNIX, JavaScript and HTML</p></li>
<li><p>Documentation: LaTeX, Markdown</p></li>
<li><p>Literate Programming: Sweave, Knitr</p></li>
<li><p>Version control: git</p></li>
<li><p>Cluster computing</p></li>
</ul>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Frederic Reinier</h2>
<div id="currently-searching-for-an-data-scientist-position" class="section level3">
<h3>Currently searching for an Data Scientist position</h3>
</div>
</div>
<div id="research-experience" class="section level2" data-icon="laptop">
<h2>Research Experience</h2>
<div id="bioinformatic-scientist" class="section level3">
<h3>Bioinformatic Scientist</h3>
<p>HalioDX</p>
<p>Marseille, France</p>
<p>2018 - today</p>
<ul>
<li>Single-Cell RNA-seq Analysis for Academic and Pharma Customers.</li>
<li>Data anlysis on Genomic data for Gene expression Platform and correlation analysis with ImmunoHistoChemistry (Protein) analysis.</li>
</ul>
</div>
<div id="bioinformatician-data-scientist" class="section level3">
<h3>Bioinformatician, Data scientist</h3>
<p>Inserm, UMR_S-1065, C3M</p>
<p>Nice, France</p>
<p>2016 - 2018</p>
<ul>
<li>Single-Cell RNA-Seq study on uveal melanoma: Identification of molecular pathways triggered by gene expression on heterogeneous cell types in tumors.</li>
<li>Bioinformatics analysis for projects in the network of Cancéropôle PACA.</li>
<li>Strong interaction with scientists. Definition of Data analysis pipeline according to their requirements and objectives, principally on cancer genomics.</li>
</ul>
</div>
<div id="visiting-scientist" class="section level3">
<h3>Visiting Scientist</h3>
<p>University of Washington, Dept of Genome Sciences</p>
<p>Seattle, USA</p>
<p>2013 - 2016</p>
<ul>
<li>Investigated as data scientist on mendelian genomics disease. Member of Center for mendelian Genomics, UWCMG.</li>
<li>Developped pipeline analysis for large indels on whole exome data.</li>
<li>Detected POLD1 mutation in Progeria sample genome.</li>
</ul>
</div>
<div id="computer-scientist-and-data-analyst" class="section level3">
<h3>Computer Scientist and Data Analyst</h3>
<p>Center of Research and Development in Sardinia</p>
<p>Pula, Italy</p>
<p>2006 - 2012</p>
<ul>
<li>Investigated as data scientist on “1000 Sardinian Genomes”, for the identification of genomic variation on the specific sardinian population. In charge on Data analysis from sequencing machines.</li>
<li>Developped pipeline analysis for whole genome and exome sequence data.</li>
<li>Collaboration with international consortium.</li>
</ul>
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="postgraduate-degree-in-bioinformatics" class="section level3">
<h3>Postgraduate Degree in Bioinformatics</h3>
<p>University of Mediterranean Luminy</p>
<p>Marseille, France</p>
<p>2005</p>
<p>Thesis: Integration of genomic information to improve multiple sequence alignments.</p>
</div>
<div id="msc-in-computer-science" class="section level3">
<h3>MSc in Computer Science</h3>
<p>University of Mediterranean Luminy</p>
<p>Marseille, France</p>
<p>2004</p>
</div>
</div>
<div id="selected-publications-and-posters" class="section level2" data-icon="file">
<h2>Selected Publications and Posters</h2>
<div id="copy-number-changes-identified-using-whole-exome-sequencing-in-nonsyndromic-cleft-lip-and-palate-in-a-honduran-population" class="section level3">
<h3>Copy Number Changes Identified Using Whole Exome Sequencing in Nonsyndromic Cleft Lip and Palate in a Honduran Population</h3>
<p>Birth Defects Res.</p>
<p>Seattle, USA</p>
<p>2017</p>
<p>Cai Y, Patterson KE, <strong>Reinier F</strong> et al.</p>
</div>
<div id="mandibular-hypoplasia-deafness-progeroid-features-and-lipodystrophy-mdpl-syndrome-in-the-context-of-inherited-lipodystrophies." class="section level3">
<h3>Mandibular hypoplasia, deafness, progeroid features and lipodystrophy (MDPL) syndrome in the context of inherited lipodystrophies.</h3>
<p>Metabolism. 2015 Nov;64.</p>
<p>Seattle, USA.</p>
<p>2015</p>
<p><strong>Reinier F</strong>, Zoledziewska M, Hanna D et al.</p>
</div>
<div id="protogene-turning-amino-acid-alignments-into-bona-fide-cds-nucleotide-alignments." class="section level3">
<h3>PROTOGENE: turning amino acid alignments into bona fide CDS nucleotide alignments.</h3>
<p>NAR 2006 July 1; 34 PMID: 16845080.</p>
<p>Marseille, FRANCE.</p>
<p>2006</p>
<p>Moretti S, <strong>Reinier F</strong> et al.</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    s = el.dataset['icon'] || 'bookmark';
    el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
